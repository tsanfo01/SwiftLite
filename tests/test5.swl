class Fruit {
    var apple:Int = 5;
    var banana:Int = 3;

    init(apple:Int, banana:Int) {
        apple = self.apple;
        banana = self.banana;
    }

    func count (apple:Int, banana:Int) -> String {
        return apple + banana;
    }
}

var fruitPlate:Fruit = Fruit(8,9);
fruitPlate.count(10,12);