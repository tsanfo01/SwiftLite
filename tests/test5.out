class Fruit {
var apple: Int = 5;
var banana: Int = 3;init(banana: Int, apple: Int) {
banana = self.banana;
apple = self.apple;
}
func count(apple: Int, banana: Int) -> String{
return apple + banana;
}
}
var fruitPlate: Fruit = Fruit(9, 8);
fruitPlate.count(12, 10);